_src/css/_global.scss

```diff
@@ -0,0 +1,29 @@
+html,
+body {
+  min-height: 100%;
+  height: 100%;
+}
+
+body {
+  width: 100%;
+  margin: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Oswald', sans-serif;
+}
+
+p {
```

Comment:
@felipprodrigues The font-family used here must be on `body` instead cause you can miss other elements like `span`, `a`, `strong`, etc.